ibazel/command/command.go

```diff
@@ -41,37 +37,18 @@ type Command interface {
 
 // start will be called by most implementations since this logic is extremely
 // common.
-func start(b bazel.Bazel, target string, args []string) (*bytes.Buffer, *exec.Cmd) {
-	var filePattern strings.Builder
-	filePattern.WriteString("bazel_script_path")
-	if runtime.GOOS == "windows" {
-		filePattern.WriteString(".bat")
-	}
-	
-	tmpfile, err := ioutil.TempFile("", filePattern.String())
-	if err != nil {
-		fmt.Print(err)
-	}
-	// Close the file so bazel can write over it
-	if err := tmpfile.Close(); err != nil {
-		fmt.Print(err)
-	}
-
-	// Start by building the binary
-	_, outputBuffer, _ := b.Run("--script_path="+tmpfile.Name(), target)
-
-	runScriptPath := tmpfile.Name()
-
-	// Now that we have built the target, construct a executable form of it for
-	// execution in a go routine.
-	cmd := execCommand(runScriptPath, args...)
+func start(target string, args []string) (*bytes.Buffer, *exec.Cmd) {
+	// Build and run the target in a go routine with bazel. Since the direct_run
+	// functionaliy was made default that works fine.
+	args = append([]string{"run", target}, args...)
+	cmd := execCommand(*bazel.BazelPath, args...)
 	cmd.Stdout = os.Stdout
 	cmd.Stderr = os.Stderr
 
 	// Set a process group id (PGID) on the subprocess. This is
 	cmd.SysProcAttr = &syscall.SysProcAttr{Setpgid: true}
 
-	return outputBuffer, cmd
+	return nil, cmd
```

Comment:
I think that by not returning the buffer [`output_runner`](https://github.com/bazelbuild/bazel-watcher/tree/master/ibazel/output_runner) is not going to be able to work. Can you help me understand why that isn't the case if you believe so?

Reply:
Ah yeah that is a good point. I have only tried using it once. Did not immediately work and have not tried since. Also since autogazelle was released it is not that important for now for our bazel uses. Either way it should not break. Will also take a look.